tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -484,6 +489,16 @@ def test_pdb_check_simultaneous_node_drains(
         time.sleep(30)
         try:
             drain_nodes([node_B])
+            # Check Mon failovers in 10mins, 15mins and 20 mins
+            timeout = [600, 300, 300]
+            for failover in timeout:
+                sample = TimeoutSampler(
+                    timeout=failover,
+                    sleep=10,
+                    func=helpers.check_mon_pods_eq_3,
+                )
+                if sample.wait_for_func_status(result=True):
```

Comment:
I am not sure if I understand this correctly, why are we iterating here for every value in the timeout list, instead cant we directly set the timeout to 1200 and see if we are seeing any additional mon pod? If we cover 1200sec that should even cover 600sec and 900sec  unless there is some difference in the mon failover behaviour.

Reply:
The testcase goes like this to check the mon failover behaviour in 10th min and 15th min and 20th min, This is as per https://bugzilla.redhat.com/show_bug.cgi?id=1946573#c8

Reply:
The default mon failover is for 10 mins only but in cases of drain the rook operator doubles the timeout to 20 mins, Hence to have a constant check on mon failovers we do it in 10th minute and 15 and then finally 20th minute as suggested by Santosh Pillai in BZ  https://bugzilla.redhat.com/show_bug.cgi?id=1959964#c5
Testcase: https://bugzilla.redhat.com/show_bug.cgi?id=1959980#c8
Approved Testplan https://docs.google.com/document/d/1vzjJFai0UQCIsN0RxewkmpMsLZUj8FSDx8OodrAkxro/edit#heading=h.p6t74oxeb8qr

Reply:
Yea, that comment statement confused me.